src/Symfony/Component/Form/Extension/Validator/ValidatorExtension.php

```diff
@@ -36,7 +36,7 @@ public function __construct(ValidatorInterface $validator, bool $legacyErrorMess
     {
         $this->legacyErrorMessages = $legacyErrorMessages;
 
-        $metadata = $validator->getMetadataFor('Symfony\Component\Form\Form');
+        $metadata = $validator->getMetadataFor(\Symfony\Component\Form\Form::class);
```

Comment:
I think it's fine to import this class.

Reply:
`ValidatorExtension` already uses `Symfony\Component\Form\Extension\Validator\Constraints\Form;` so we would need to alias one.

Reply:
All right, let's keep it that way then.